app/services/scraper.rb

```diff
@@ -37,48 +32,54 @@ def scrape_for_new_tournaments
     log_scrape(:tournament)
   end
 
-  def scrape_data_from_all_tournaments
+  def scrape_tournament_data
     return if data_scraped_recently?
 
     Tournament.with_incomplete_data.each do |tournament|
-      scrape_tournament(tournament)
+      calculate_correlations_for(tournament)
     end
 
     log_scrape(:data)
   end
 
-  def scrape_tournament_series(pga_id)
-    Tournament.where(pga_id: pga_id).each do |tournament|
-      scrape_tournament(tournament)
+  def calculate_correlations_for(tournament)
```

Comment:
@spalmer0 this method is the most important chunk out of this entire PR - it's where we scrape/parse the relevant pages, calculate the correlations, and save a new `Correlation` record